src/connection_manager.rs

```diff
@@ -64,7 +64,7 @@ impl std::default::Default for OperationRetryOptions {
     fn default() -> Self {
         OperationRetryOptions {
             operation_timeout: Duration::from_secs(30),
-            retry_delay: Duration::from_millis(500),
+            retry_delay: Duration::from_secs(5),
```

Comment:
from_millis(500) is 500 ms which is 0.5 seconds

Reply:
Yes, I wanted to increase the default delay.